transaction/std/src/transaction_builder.rs

```diff
@@ -316,7 +322,12 @@ impl<FPR: FogPubkeyResolver> TransactionBuilder<FPR> {
 
     /// Gets the transaction fee.
     pub fn get_fee(&self) -> u64 {
-        self.fee
+        self.fee.value
+    }
+
+    /// Gets the transaction fee token id.
+    pub fn get_fee_token_id(&self) -> TokenId {
```

Comment:
Can we afford to make `get_fee` return `Amount`? I'm a bit concerned about people not noticing/using this method.

Reply:
yeah we could probably do that

Reply:
done